packages/manager/src/features/Firewalls/FirewallDetail/Devices/FirewallDeviceLanding.tsx

```diff
@@ -44,9 +43,7 @@ export const FirewallDeviceLanding = React.memo(
     const routeMatch = useRouteMatch();
     const location = useLocation();
 
-    const flags = useFlags();
-    const entityName = flags.firewallNodebalancer ? 'services' : 'Linodes';
-    const helperText = `Assign one or more ${entityName} to this firewall. You can add ${entityName} later if you want to customize your rules first.`;
+    const helperText = `Assign one or more services to this firewall. You can add serivces later if you want to customize your rules first.`;
```

Comment:
ticks and typo while we're here 😄 

```suggestion
    const helperText = "Assign one or more services to this firewall. You can add services later if you want to customize your rules first.";
```